### PR TITLE
SLING-11155: add a test to make sure we bind proxy resource type registrations by full path only

### PR DESCRIPTION
**Pull request by karlpauls**

### CODE REVIEW

File: it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
 import org.apache.sling.api.resource.ResourceResolverFactory;
 import org.junit.Test;
 
+import static junit.framework.TestCase.assertFalse;
```
Comment (raducotescu) [inline it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java]: Let's not mix JUnit versions.

File: it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java [hunk 2]
```diff
@@ -114,4 +115,18 @@ public void testAppsHello() throws LoginException {
             }
         }
     }
+
+    @Test
+    public void testRt() throws LoginException {
```
Comment (raducotescu) [inline it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java]: Shouldn't this test be in a different Test Suite? Either a new one or https://github.com/apache/sling-org-apache-sling-scripting-bundle-tracker-it/blob/master/it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundlePrecompiledTeleportedIT.java?

**Review by raducotescu — CHANGES_REQUESTED** [review]
LGTM, though I'd like some minor changes.

File: it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java (hunk 1 above)
Comment (karlpauls) [reply it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java]: Done

File: it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java (hunk 2 above)
Comment (karlpauls) [reply it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java]: Moved

**Review by raducotescu — APPROVED** [review]

**Pull request merged**